packages/shared/src/components/auth/PostAuthBanner.tsx

```diff
@@ -1,11 +1,116 @@
 import React, { ReactElement } from 'react';
+import { useSearchParams } from 'next/navigation';
+import { useQuery } from '@tanstack/react-query';
+import classNames from 'classnames';
 import { feature } from '../../lib/featureManagement';
 import { checkIsBrowser, checkIsExtension, UserAgent } from '../../lib/func';
 import { AuthExtensionBanner } from './AuthExtensionBanner';
 import { AuthenticationBanner } from './AuthenticationBanner';
+import { getBasicUserInfo } from '../../graphql/users';
+import { OnboardingHeadline } from './OnboardingHeadline';
+import { sizeClasses } from '../ProfilePicture';
+import { Image, ImageType } from '../image/Image';
 import { useConditionalFeature } from '../../hooks';
+import { IconSize } from '../Icon';
+import {
+  getSocialReferrer,
+  socialCTA,
+  socialGradient,
+  socialIcon,
+  type SocialIconType,
+} from '../../lib/socialMedia';
+import { capitalize } from '../../lib/strings';
+import { useAuthContext } from '../../contexts/AuthContext';
+
+const geoToEmoji = (geo: string): string => {
```

Comment:
It's fine for now, but we could have moved it to lib function assuming product might want this in more places

Reply:
I'll move it, no problem!

Reply:
@rebelchris Moved, and also moved "geoToCountry" and added an extra check on it. Realized there's quite a few countries with "The" prefix, such as The United States, The Netherlands, The United Kingdom etc. 

old:
![Screenshot 2024-11-25 at 10 57 55](https://github.com/user-attachments/assets/aa24100f-061b-4bd7-a435-5809e7ec990e)

new: 
![Screenshot 2024-11-25 at 10 45 47](https://github.com/user-attachments/assets/0048af07-dc7d-4217-969a-a28db033d861)

Reply:
@AmarTrebinjac I think changes have not been pushed yet.

Reply:
Ah, it has been pushed, just this one left.